glean_parser/metrics.py

```diff
@@ -189,7 +192,7 @@ class TimeUnit(enum.Enum):
     millisecond = 2
     second = 3
     minute = 4
-    hours = 5
+    hour = 5
```

Comment:
Is this sneaking in from another branch?

Reply:
No -- just a bug found in the process of writing this PR...